react-assignment1/src/components/ProfileCard/ProfileCard.css

```diff
@@ -0,0 +1,8 @@
+.card{
+    border: 1px solid gray;
+}
+
+.img{
```

Comment:
use better classnames that are different than html tag names
```suggestion
.img {
```